gradle.properties

```diff
@@ -0,0 +1,5 @@
+org.gradle.daemon=true
+org.gradle.caching=true
+org.gradle.parallel=true
+org.gradle.configureondemand=true
```

Comment:
This seems like another one that is fine for your own ~/.gradle/gradle.properties. It's incubating and it is unlikely to help much when daemon's already enabled. But if it does help Travis, we can enable it in .travis.yml.